doc/release-notes/0.8/0.8.md

```diff
@@ -0,0 +1,76 @@
+-->
+
+# Eclipse OpenJ9 version 0.8 release notes
+
+## Supported platforms
+
+OpenJDK binaries that contain the Eclipse OpenJ9 virtual machine are available from the
+[AdoptOpenJDK community](https://adoptopenjdk.net/?variant=openjdk8-openj9). These builds are
+being tested against the OpenJDK test suite.
+
+The following platforms are available:
+
+|Operating system | Platform architecture | Comments                                    |
+|-----------------|-----------------------|---------------------------------------------|
+|Linux            | x86, IBM Power, IBM Z | Windows subsytem for Linux is not supported |
+|AIX              | IBM Power             | Untested                                    |
+|Windows          | x86                   | Untested                                    |
+
+
+## Known Issues
+
+The v0.8 release contains the following known issues and limitations:
+
+#### Direct dump reader
+[Issue #378](https://github.com/eclipse/openj9/issues/378):
+The direct dump reader is currently not working, which has an impact on diagnosing problems with the VM, Garbage Collector, and the JIT. For example, [issue #968](https://github.com/eclipse/openj9/issues/968).
+
+#### Signal handling
+[Issue #54](https://github.com/eclipse/openj9/issues/54): Some signals are not being handled correctly by the VM. In particular, for CTRL-C or other signals that shutdown the VM, the shutdown hooks are not running. See [Issue #378](https://github.com/eclipse/openj9/issues/378)
+
+#### Large Java heaps
+[Issue #479](https://github.com/eclipse/openj9/issues/479): The binaries available at AdoptOpenJDK are built only with the compressed references VM, which does not support Java heaps larger
+than 57 GB (or ~62 GB with command line options). When builds that contain the non-compressed references VM are available, this limitation will be removed. If
+you want to build a non-compressed references VM yourself, you can do so now by following our [detailed build instructions](https://github.com/eclipse/openj9/blob/master/buildenv/Build_Instructions_V8.md).
+
+#### Graphics Processor Unit (GPU) support
+GPU acceleration with `java.util.Arrays.sort()` is not supported, although `com.ibm.gpu.Maths` can be used directly.
+
+#### Functional verification tests
+The following issues relate to test failures:
+
+- [#238](https://github.com/eclipse/openj9/issues/238), [#252](https://github.com/eclipse/openj9/issues/252), [#244](https://github.com/eclipse/openj9/issues/244), [#362](https://github.com/eclipse/openj9/issues/362), [#366](https://github.com/eclipse/openj9/issues/366), [#480](https://github.com/eclipse/openj9/issues/480), [#1128](https://github.com/eclipse/openj9/issues/1128).
+
+Because some tests are not yet working, functional problems might not be exposed. In addition, some IBM authored functional verification testing is not yet contributed to Eclipse, such as issue  [#724](https://github.com/eclipse/openj9/issues/724).
```

Comment:
I personally would reword this as "Because some tests are not yet enabled" rather than not yet working, if you are referring to the tests not yet migrated to the OpenJ9 repo.  As for the next sentence, is it true that we have traceformat tests that have not been contributed?  We will have to track it down, as tests for utilities was not owned by our team previously.

Reply:
Done